js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/Node.tsx

```diff
@@ -109,6 +110,13 @@ export const Node = ({
   const showArrow =
     !isAssetNode || (viewType === 'tree' && downstream.filter((id) => graphData.nodes[id]).length);
 
+  const ref = React.useRef<HTMLButtonElement | null>(null);
+  React.useLayoutEffect(() => {
+    if (ref.current && isSelected && !isElementInsideSVGViewport(document.activeElement)) {
+      ref.current.focus();
+    }
+  }, [isSelected]);
+
```

Comment:
Can you add a comment here identifying the specific cases where this happens?

It's a bit unusual because if someone had clicked the item to select it, the element would already be focused from the click. If the page has just loaded, I think both the asset graph and this will fight for the focus and it's unclear which will fire first.  Definitely would need instructions for reproing the case that's covered by this hook :-)

Reply:
This logic basically checks if the svg viewport has focus currently and if it does then it doesnt take it away. this way you can still use arrow keys in the svg viewport